.github/workflows/reusable-build-test-release.yml

```diff
@@ -783,6 +789,39 @@ jobs:
           path: |
             *_markdown.txt
 
+  appinspect-api:
+    name: appinspect api ${{ matrix.tags }}
+    needs: build
+    if: |
+      always() &&
+      needs.build.result == 'success'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        tags:
+          - "cloud"
+          - "self-service"
+          - ""
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: package-splunkbase
+          path: build/package
+      - name: AppInspect API
+        uses: splunk/appinspect-api-action@v3.0
+        with:
+          username: ${{ secrets.SPL_COM_USER }}
+          password: ${{ secrets.SPL_COM_PASSWORD }}
+          app_path: build/package/
```

Comment:
Here you can use output of download-artifact action: https://github.com/actions/download-artifact#download-path-output